app/src/main/java/com/kks/nimblesurveyjetpackcompose/viewmodel/splash/SplashViewModel.kt

```diff
@@ -14,18 +18,58 @@ import javax.inject.Inject
 
 @HiltViewModel
 class SplashViewModel @Inject constructor(
+    private val loginRepo: LoginRepo,
     @IoDispatcher private val ioDispatcher: CoroutineDispatcher,
 ) : ViewModel() {
-
+    private val _shouldShowLoading = MutableStateFlow(false)
     private val _shouldNavigateToLogin = MutableStateFlow(false)
+    private val _isLoginSuccess = MutableStateFlow(false)
+    private var _error = MutableStateFlow<ErrorModel?>(null)
+
+    val getError: StateFlow<ErrorModel?>
```

Comment:
Not sure if I missed it before. But making it `error` would be nicer.
```suggestion
    val error: StateFlow<ErrorModel?>
```

Reply:
I have renamed it in [Home Screen PR](https://github.com/kaungkhantsoe/NimbleSurveyJetpackCompose/pull/74)